crates/swc_ecma_parser/src/parser/expr.rs

```diff
@@ -267,21 +267,6 @@ impl<I: Tokens> Parser<I> {
                     })));
                 }
 
-                tok!("import") => {
-                    let import = self.parse_ident_name()?;
-
-                    if is!(self, '.') {
```

Comment:
This was a red herring and didn't work. For example, this does `is!(self, '.')` then `expect!(self, '.')` below.